source/beamline/BioXAS/BioXASSideMonochromator.cpp

```diff
@@ -31,6 +31,10 @@ BioXASSideMonochromator::BioXASSideMonochromator(QObject *parent) :
 	regionAStatus_ = new AMReadOnlyPVControl("RegionAStatus", "BL1607-5-I22:Mono:Region:A", this);
 	regionBStatus_ = new AMReadOnlyPVControl("RegionBStatus", "BL1607-5-I22:Mono:Region:B", this);
 
+	hcControl_ = new AMReadOnlyPVControl("hc Constant", "BL1607-5-I22:Energy:EV:fbk:tr.A", this);
```

Comment:
I just want to double check that the PV names are supposed to be named ```...:fbk:tr.A```.  That is a convention I have never seen before.  Do you know what the ```tr.A/B/K``` is supposed to imply?